### ISSUE TITLE
Cannot Access Munin Apache

### ISSUE THREAD

**Issue by jordan-josephs** [munin-monitoring/munin#1463]
I am running Munin master on a remote server running Ubuntu 20.04. Following the instructions I have edited the munin.conf file and apache24.conf file accordingly. I then symlinked the apache24.conf file to /etc/apache2/conf-enabled and navigated to http://domain_name/munin but I get greeted with -

Forbidden
You don't have permission to access this resource.

I have changed the permissions in apache24.conf as instructed but cannot seem to access munins webpage at all.

**Comment by sumpfralle** [comment]
I guess, the section `Allow non-local access to web interface` in the munin documentation of the package (`/usr/share/doc/munin/README.Debian.gz`) will help you solve this issue.

**Comment by jordan-josephs** [comment]
I did change Require Local to Require all granted. Even with that change at the very beginning of my install I have this issue. Sorry it took so long for my response. I was out of town.

**Comment by sumpfralle** [comment]
Could you post your content of `/etc/munin/apache2.conf` somewhere?

**Comment by jordan-josephs** [comment]
Sure here you go - https://termbin.com/h3vh
But isn't the file name apache24.conf?

**Comment by sumpfralle** [comment]
>  But isn't the file name apache24.conf?

Indeed - you passed the test :)

Just a stupid idea: did you restart `apache2` after changing the configuration?

Do you see any messages in apache's log?

**Comment by jordan-josephs** [comment]
Yeah I did restart apache. And no weird logs in apaches log file. Just standard output.

**Comment by sumpfralle** [comment]
The apache log should contain the request, which it rejected as "Forbidden". This log line will contain a more verbose reason for the rejection (maybe in case of a numeric identifier). Does this help?
